runtime/README.md

```diff
@@ -0,0 +1,488 @@
+
+| **Name** | **Type** | **Description**
+| -------- | -------- | ---------------
+| crc      | u32      | CRC over other output arguments, computed by Caliptra. Little endian.
+| result   | u32      | Result code. Little endian.
+
+### STASH\_MEASUREMENT
+
+Make a measurement into the DPE default context. This command is intendend for
+callers who update infrequently and cannot tolerate a changing DPE API surface.
+
+Internally, this will call the DPE DeriveChild command.
+
+Command Code: `0x4D45_4153` ("MEAS")
+
+Table: `STASH_MEASUREMENT` input arguments
+
+| **Name**     | **Type** | **Description**
+| --------     | -------- | ---------------
+| crc          | u32      | CRC over other input arguments, computed by the caller. Little endian.
+| metadata     | u8[4]    | 4-byte measurement identifier.
+| measurement  | u8[48]   | Data to measure into DPE.
+
+
+Table: `STASH_MEASUREMENT` output arguments
+
+| **Name**    | **Type** | **Description**
+| --------    | -------- | ---------------
+| crc         | u32      | CRC over other output arguments, computed by Caliptra. Little endian.
+| result      | u32      | Result code. Little endian.
+| dpe\_result | u32      | Result code of DPE DeriveChild command. Little endian.
+
+### INVOKE\_DPE\_COMMAND
+
+Command Code: `0x4450_4543` ("DPEC")
+
+Table: `INVOKE_DPE_COMMAND` input arguments
+
+| **Name**     | **Type**      | **Description**
+| --------     | --------      | ---------------
+| crc          | u32           | CRC over other input arguments, computed by the caller. Little endian.
+| data         | u8[...]       | DPE command structure as defined in the DPE iRoT profile
+
+
+Table: `INVOKE_DPE_COMMAND` output arguments
+
+| **Name**    | **Type**      | **Description**
+| --------    | --------      | ---------------
+| crc         | u32           | CRC over other output arguments, computed by Caliptra. Little endian.
+| result      | u32           | Result code. Little endian.
+| data        | u8[...]       | DPE response structure as defined in the DPE iRoT profile.
+
+## CRC
+
+For every command input/output arguments which have a "crc" field, the request
+and response feature a checksum. This mitigates glitches between clients
+and Caliptra.
+
+The CRC is a little-endian 32-bit value, defined as:
+
+```
+0 - (SUM(command code bytes) + SUM(request/response bytes))
+```
+
+The sum of all bytes in a request/response body, and command code, should be
+zero.
+
+If Caliptra detects an invalid CRC in input parameters, it will return `BAD_CRC`
+as the result.
+
+Caliptra will also compute a CRC over all responses and write it to the crc
+field.
+
+## Runtime Firmware Updates
+
+Caliptra Runtime firmware accepts impactless updates which will update
+Caliptra’s firmware without resetting other cores in the SoC.
+
+### Applying Updates
+
+A Runtime Firmware update is triggered by the `CALIPTRA_FW_LOAD` command. Upon
+receiving this command, Runtime Firmware will:
+
+1. Write-lock mailbox
```

Comment:
There'll be more to do this because of multi-pass -- LMS will make the payload larger than the mailbox. But ok to tackle in a subsequent PR

Reply:
Added to opens